rocker-test-template-java-17/pom.xml

```diff
@@ -0,0 +1,147 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>rocker-test-template-java-17</artifactId>
+    <name>rocker-test-template-java-17</name>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>com.fizzed</groupId>
+        <artifactId>rocker</artifactId>
+        <version>2.0.2-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+		        <java.version>17</java.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>com.fizzed</groupId>
+                <artifactId>rocker-maven-plugin</artifactId>
+                <version>${project.version}</version>
+                <executions>
+                    <execution>
+                        <id>generate-rocker-templates</id>
+                        <phase>generate-test-sources</phase>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <javaVersion>${java.version}</javaVersion>
+                            <templateDirectory>${basedir}/src/test/java</templateDirectory>
+                            <outputDirectory>${basedir}/target/generated-test-sources/rocker</outputDirectory>
+                            <discardLogicWhitespace>false</discardLogicWhitespace>
+                            <addAsTestSources>true</addAsTestSources>
+                            <optimize>true</optimize>
```

Comment:
Can we try out a new feature that snuck into the last release?  It'd just require adding the ```<plainTextStrategy>STATIC_BYTE_ARRAYS</plainTextStrategy>``` as a configuration to the rocker-maven-plugin?  It generates a slightly different loading strategy for plain text and it'd be a nice to have unit test.

Reply:
done